### DISCUSSION TITLE
Build fails

### DISCUSSION THREAD

**Discussion by WS-Snyder** [YosysHQ/yosys#3272] [Q&A]
cloned repo today; and build fails at last step [100%]
[build.log](https://github.com/YosysHQ/yosys/files/8439321/build.log)
Build log (attached) shows a problem linking (because of 'readline') library(s)
-nothing is shown as missing, but I don;t understand the references to
duplicate items.

**Comment by WS-Snyder** [comment]
Some aux info: Used 'apt install' to load all the requisite packages and OS is Ubuntu 20.04 LTS.

**Reply by mmicko** [reply]
According to log you have readline on two locations (one from apt and one locally installed) that are in conflict and that produce linking error.

**Comment by WS-Snyder** [comment]
Re: two readline installs.

Your conjecture is probably correct; but why is the MakeFile
setup so that there is a conflict; ie. why is not cmake used?

Thanks
Warren
